client/src/generated/index.tsx

```diff
@@ -302,7 +305,7 @@ export type EventsQuery = { __typename?: 'Query' } & {
   events: Array<
     { __typename?: 'Event' } & Pick<
       Event,
-      'id' | 'name' | 'canceled' | 'description' | 'capacity'
+      'id' | 'name' | 'canceled' | 'description' | 'url' | 'capacity'
```

Comment:
Would it make sense to have these ordered alphabetically?
(The OCD in me. Would not block a merge IMHO).